sklearn/mixture/tests/test_gmm.py

```diff
@@ -111,7 +121,11 @@ def test_lmvnpdf_full():
     reference = _naive_lmvnpdf_diag(X, mu, cv)
     lpr = mixture.log_multivariate_normal_density(X, mu, fullcv, 'full')
```

Comment:
This function is testing for both functionality and whether the deprecation warning is being raised. So that line is a call to the function, which should raise the deprecation warning. However, we are not testing for the deprecation warning until L124.